lib/ts/utils.ts

```diff
@@ -225,3 +226,7 @@ export async function removeServerlessCache() {
         await removeFile(tempFilesPath[i]);
     }
 }
+
+export function frontendHasInterceptor(req: Request): boolean {
```

Comment:
This should not be in the PR. But it's there since the base was wrong. So please correct.